src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/README.md

```diff
@@ -0,0 +1,31 @@
+I'm a class to control the ultrasonic HCSR-04 sensors.
+You can use many ultrasonic sensors at the same time. Just change the PINs id. 
+
+To use:
+(inspector)
+ultrasonic := board installDevice: PotHCSR04Device new.
```

Comment:
Continue my proposal to avoid default device configuration:
In inspector it will also quite simple expression:
```Smalltalk
ultrasonic := board installDevice: (PotHCSR04Device triggerPin: gpio5 echoPin: gpio17).
```

Reply:
I change this in the last PR